### ISSUE TITLE
Suggestion in how to train the network

### ISSUE THREAD

**Issue by KAOeric** [DeepanChakravarthiPadmanabhan/KeypointNet#2]
Thanks for your wonderful sharing first! Recently I'm trying to train keypointnet using the code in tensorflow. But the output predicted images are terrible. And I saw the result showed in your presentation is much more better than mine. So I want to know do your have any suggestion in how to train the network especially the value of the parameters like batch_size, learning_rate, the weight of each loss, etc.Thanks for your attention.
